weaviate/client.py

```diff
@@ -245,7 +238,7 @@ def __init__(
             proxies=config.proxies,
             trust_env=config.trust_env,
         )
-        self._connection.connect(skip_init_checks)
+        # self._connection.connect(self.__skip_init_checks)
```

Comment:
so we force users to call connect?

Reply:
We will have to once we add an async connection to the client since you cannot have an `async def __init__` function. So I'm making this change here so that the future change is less breaking

Alternatively, we could have a `WeaviateClient` and `WeaviateClientAsync` but this would lead to a huge amount of code duplication IMO, which we should avoid if we can

Reply:
```python
client = await weaviate.connect_to_wcs()

async def connect_to_wcs():
    client = WeaviateClient(...)
    await client.connect()
    return client
```

Reply:
oki, do you want to remove the commented out code?